### PR TITLE
Redesign/external website

### LINKED ISSUE #586: Implement Laura's design ideas for main page of the external part of the website
Ideas to implement:
- Order the activity cards by date top-bottom left-right instead of left-right top-bottom
- Change the font color of the location name on the activity cards to white
- Change the navbar items to: Over ons (dropdown), Word lid!, Word oud-lid!, Artikelen, Foto-albums, Sponsoring
- Get rid of light-grey navbar halfway on the page
- On main banner with logo, add text "Lid worden bij Alpha? Dat kan op een van onze activiteiten! Laat hier je mail achter en we nemen contact met je op!" below the logo plus the "Ik heb interesse!" button. Potentially also darken the background image to make the new text more readable.
- Make potential member section a lighter background color

### PR DESCRIPTION
**Pull request by Ellen-Wittingen**
fix #586 

I changed all the things mentioned in #586. See the screenshot. I didn't change the sidebar navbar to match the top navbar in terms of menu items as that one seems to be very tied into the one you have when you are logged in, so I will only mess with that one if we really want it, I think its fine for now though. The "over ons" dropdown contains all public static pages except the sponsoring, word lid, and word oud-lid pages as those are on the navbar already.

image

### CODE REVIEW

**Comment by wilco375** [comment]
image
There is no more padding at the top of logged in pages, and I can no longer edit activities (`Attempted to resolve `file-upload`, which was expected to be a component, but nothing was found.`)

File: app/components/header-nav.js [hunk 1]
```diff
@@ -61,5 +79,29 @@ export default Component.extend({
         localStorage.setItem('locale', 'nl');
       }
     },
+    setAvailableStaticPages() {
+      if (!this.session.isAuthenticated && !this.media.isMobile) {
+        this.store.findAll('static-page').then((pages) => {
+          let newAvailableStaticPages = {};
+          // make key-value pairs for all found static pages
+          pages.forEach(
```
Comment (wilco375) [inline app/components/header-nav.js:87]: Could be more compactly written as `let newPages = pages.reduce((obj, page) => Object.assign(obj, { [page.id]: page.title }), {})`

File: app/styles/components/index/public/activities.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+@include media-breakpoint-up(md) {
+    .activity-list {
+        column-count:2;
```
Comment (wilco375) [inline app/styles/components/index/public/activities.scss:3]: missing spaces after `:`, not sure why the linter doesn't pick this up

File: app/templates/application.hbs [hunk 3]
```diff
@@ -14,7 +14,7 @@
         {{if layoutManager.leftSideBarOpen "left-sidebar-open"}}'
     >
       <div class='row justify-content-center'>
-        <div class='col-12 col-md-8 col-lg-9 col-xl-10'>
+        <div class='col-12 col-md-10 col-lg-9 col-xl-10'>
```
Comment (wilco375) [inline app/templates/application.hbs:17]: Are you sure this doesn't affect the layout of any other pages?

**Comment by wilco375** [comment]
image
I don't think you changed anything here, but this might look better when everything is aligned to the left on mobile

**Comment by guidojw** [comment]
> image
> 
> There is no more padding at the top of logged in pages, and I can no longer edit activities (`Attempted to resolve `file-upload`, which was expected to be a component, but nothing was found.`)

Did you run yarn install after #570 and #589?

**Comment by Ellen-Wittingen** [comment]
> > image
> > There is no more padding at the top of logged in pages, and I can no longer edit activities (`Attempted to resolve `file-upload`, which was expected to be a component, but nothing was found.`)
> 
> Did you run yarn install after #570 and #589?

This branch is behind on staging, so those fixes are not in here yet, Ill do that today

File: app/templates/application.hbs (hunk 3 above)
Comment (Ellen-Wittingen) [reply app/templates/application.hbs:17]: In my head I thought it was weird that it went like: 12 8 9 like md randomly is smaller, so I changed it to be 12 10 9, so as the screen gets bigger it takes up less percent of the space. But I didnt see xl-10, and with that observation my change does not make any sense. So Im changing it back, better be safe than sorry.

**Comment by Ellen-Wittingen** [comment]
I implemented / fixed all of Wilco's remarks. I also merged the most recent version of the staging branch into here, so activities can be created/edited again.

**Review by wilco375 — APPROVED** [review]

**Pull request merged**